shcomplete/repos.py

```diff
@@ -62,14 +61,14 @@ def to_raw_urls(urls):
     github.com into raw.githubusercontent.com.
     """
     base = "https://raw.githubusercontent.com"
```

Comment:
Please put some explanation about `https://raw.githubusercontent.com`.
For me it's not clear why you need to do it.
BTW, it should be a constant defined outside of a function.